posts/project-mycorrhiza/index.md

```diff
@@ -0,0 +1,188 @@
+---
+title: "Packit dashboard and Project Mycorrhiza"
+date: 2024-10-25T10:00:00
+authors:
+  - venefilyn
+  - flachman
+tags:
+  - Project Mycorrhiza
+  - dashboard
+  - ux
+  - testing-farm
+  - logs
+  - copr
+---
+
+This term has been thrown around a bit in issues here and there relating to the dashboard and
+people usually get confused, and rightfully so! Lets dive in and see what Project Myccorhiza
+is and why we chose this name.
+
+![Photo of beech trees in a forest](./img/forest.jpg "New beech leaves, Gribskov Forest in the northern part of Sealand, Denmark.")
+Photo by [Malene Thyssen](https://commons.wikimedia.org/wiki/User:Malene) - Own work, [CC BY-SA 3.0](http://creativecommons.org/licenses/by-sa/3.0/ "Creative Commons Attribution-Share Alike 3.0"), [Link](https://commons.wikimedia.org/w/index.php?curid=752225)
+
+<!--truncate-->
+
+---
+
+[TL;DR at the bottom!](#tldr)
+
+Before I explain what Project Mycorrhiza is and what it's about, lets talk about
+the Packit dashboard and why it's been getting so much love from me the past year.
+
+It's to no surprise that Packit is used within a lot of different projects around
+Fedora, which includes [Convert2RHEL](https://github.com/oamg/convert2rhel) that I'm
+a developer of. But I've always felt like Packit's dashboard has missed the mark on
+being the go-to for getting an overview for the pipeline, results, or even the
+project itself.
+
+Since I like making websites better for my own benefit, I've spending a huge amount of my own
+time the past year to gradually improve the Packit dashboard to prepare it for something
+bigger. Which is where Project Mycorrhiza comes in.
```

Comment:
```suggestion
Since I like making websites better for my own benefit, I've been spending a huge amount of my own
time the past year to gradually improve the Packit dashboard to prepare it for something
bigger. Which is where Project Mycorrhiza comes in.
```